lesson_00/Kyvon/Kyvonreadme.md

```diff
@@ -0,0 +1,31 @@
+# Learn about Kyvon Thompson
+
+# My Niche
+I'm a Delaware and New York based music video director and editor with skills in VFX, videography, photography, lighting and mixed media. This has led me to work and collab with and help develop the visual articulation of many talented artists and other creators either in Delaware or New york.
+
+## Examples of my work
+![EX1 Medium](https://github.com/KRose77/code-differently-24-q1/assets/160963028/17c4fb8d-bc5d-4d6f-81ea-7b39a3e51d88)
+
+
+[![](https://markdown-videos-api.jorgenkh.no/youtube/814SngdguwQ)](https://youtu.be/814SngdguwQ)
+### Where im from
+I'm from Hempstead, New York.
+
+### My learning styles
+I am a visual and hands-on learner meaning I'm better at learning when I'm shown a visual representation of the lesson and then trying it out for myself.
+
+#### Hobbies
+1. Mixed media art
+2. Watching movies
+3. Playing video games (Favorite game is Kingdom Hearts BTW)
+
+#### Favorite Food
+1. Birria Tacos
+2. Churros
+3. Pizza
+4. Waffles
+5. French Toast
+6. Chicken Rasta Pasta
+
+#### How you can reach me or see more of my work
+You can reach me primarily by email or LinkedIn @KyvonThompson and Rosevisualsss52@gmail.com and you can see more of my work on Instagram @rosevisualsss
```

Comment:
More than welcome to use your real email address here, but use caution. Just making sure to mention that here.